src/vulkan/vk-buffer.cpp

```
@@ -203,6 +184,8 @@ Result BufferImpl::getSharedHandle(NativeHandle* outHandle)
     const auto& api = device->m_api;
 
     // If a shared handle doesn't exist, create one and store it.
+    // Shared buffers use VMA with dedicated allocations from the shared memory pool,
+    // so m_memory is the exclusive backing memory and can be exported directly.
```

Cppcheck (2.20.0)

[style] The function 'testOnlySetKeepPagesMapped' is never used.

(unusedFunction)